README.md

```diff
@@ -218,9 +234,11 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design                                                                        |
 | <a href="https://icons8.com/l/christmas-illustrations/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184641769-0aa18a5c-2f4a-4523-9d44-fc715b62aa18.png' alt="Merry_Christmas_freemium_illustrations" /></a> | [Merry Christmas](https://icons8.com/l/christmas-illustrations/)  <br><br> ✨ Freemium <br><br>Format <br> • Large PNG<br>• SVG<br> • GIF<br> • PSD<br> • JSON<br> • After Effects<br><br> ✌️Author <br> Icons8           |
 | <a href="https://www.ls.graphics/illustrations/pof-pof"><img width='700' src='https://user-images.githubusercontent.com/74038190/184863554-c0eef9aa-022a-433d-af05-c1281aa15a89.png' alt="Pof-Pof_freemium_illustrations" /></a> | [Pof-Pof](https://www.ls.graphics/illustrations/pof-pof)  <br><br> ✨ Freemium mium<br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> ls.graphics                                           |
-| <a href="https://www.kapustin.co/seoul"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480665-4f035a53-902a-4ecd-9b37-a174e00c8183.png' alt="Seoul_freemium_illustrations" /></a> | [Seoul](https://www.kapustin.co/seoul)  <br><br> ✨ Freemium  <br><br>Format <br> • Adobe Illustrator<br> • EPS<br> • PDF<br> • SVG<br> • Small PNG<br> • Figma<br> • Sketch<br><br> ✌️Author <br> Kapustin.co |
+| <a href="https://craftwork.design/downloads/scratches/"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480665-4f035a53-902a-4ecd-9b37-a174e00c8183.png' alt="Seoul_freemium_illustrations" /></a> | [Seoul](https://www.kapustin.co/seoul)  <br><br> ✨ Freemium  <br><br>Format <br> • Adobe Illustrator<br> • EPS<br> • PDF<br> • SVG<br> • Small PNG<br> • Figma<br> • Sketch<br><br> ✌️Author <br> Kapustin.co |
```

Comment:
I think you have mistakenly changed the image URL, kindly make it as before.
URL: `https://www.kapustin.co/seoul`